reconcile/utils/terrascript_client.py

```diff
@@ -1439,7 +1441,8 @@ def populate_tf_resource_s3(self, resource, namespace_info):
                             ],
                             "Effect": "Allow",
                             "Resource": [
-                                "${aws_s3_bucket." + identifier + ".arn}"
+                                "${aws_s3_bucket." + identifier + ".arn}",
+                                "${aws_s3_bucket." + replication_bucket_identifier + ".arn}"
```

Comment:
Shorten the variable name to pass the 79 characters line length check
```suggestion
                                "${aws_s3_bucket." + dest_bucket_identifier + ".arn}"
```